options/rtdl/generic/main.cpp

```diff
@@ -460,6 +460,13 @@ void *__dlapi_open(const char *file, int flags, void *returnAddress) {
 			return nullptr;
 		}
 
+		// If the local scope is requested, but the local scope is a nullptr, use the global scope anyway.
```

Comment:
comment is a bit redundant

did you figure out why loadScope is null? is it for ld.so + libc.so + the executable itself?

Reply:
I have no clue why, and if you want I can remove the comment.

Reply:
I was thinking that this behavior is incorrect, but posix says this about flags:
> If neither RTLD_GLOBAL nor RTLD_LOCAL are specified, then the default behavior is unspecified.